src/components/Blocking/ReportDomain.js

```diff
@@ -87,7 +87,7 @@ const ReportDomain = React.forwardRef((props, ref) => {
             ))}
             <View style={styles.btn}>
               <Button onPress={onNext}>
-                <Text>Provide info on next screen</Text>
+                <Text style={styles.btnText}>Provide info on next screen</Text>
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The text style has been updated for better readability and consistency across the app. Make sure that the new styles are consistent with the overall design of your application.

```diff
-                <Text>Provide info on next screen</Text>
+                <Text style={styles.btnText}>Provide info on next screen</Text>
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->